migration/1_init.sql

```diff
@@ -1,5 +1,6 @@
--- CREATE TABLE "user" ----------------------------------------
```

Comment:
File name should be like 001_...
Zeroes should be in the very beginning.

Reply:
Fixed.